test_psa_target.py

```diff
@@ -202,7 +202,13 @@ def _execute_test():
     """
     Execute greentea runs test as specified in test_spec.json
     """
-    cmd = ["mbedgt", "--polling-timeout", "300", "-V"]
+    if not os.path.isfile("test_spec.json"):
+        logging.critical(
+            "test_spec.json is not found, please build the tests first"
+        )
+        sys.exit(1)
+
+    cmd = ["mbedgt", "--polling-timeout", "600", "-V"]
```

Comment:
We need to include `-r none`, to avoid reset on the target

Reply:
We also need a change in https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/blob/master/psa_builder.py#L91 

So that we check tools like `mbedgt`, `mbedhtrun` and `mbed cli1` are installed

Reply:
Checking `mbedgt` should be enough, because it's the one we run directly and its python package `mbed-greentea` pulls in dependencies during `pip install`.

Reply:
Done